doc/mcuboot/index-ncs.rst

```diff
@@ -0,0 +1,68 @@
+.. _mcuboot_index_ncs:
+
+MCUboot
+#######
+
+.. contents::
+   :local:
+   :depth: 2
+
+MCUboot is a secure bootloader for microcontrollers.
+
+Overview
+********
+
+The goal of MCUboot is to define a common infrastructure for the bootloader, the system flash layout on microcontroller systems, and to provide a secure bootloader that enables easy software upgrade.
+
+MCUboot is operating-system and hardware independent, and relies on hardware porting layers from the operating system it works with.
+Currently MCUboot works with the various operating systems and SDKs like `Zephyr`_ and nRF Connect SDK.
+
+MCUboot is an open governance project.
+See the `membership list <https://github.com/mcu-tools/mcuboot/wiki/Members>`_ for current members, and the `project charter <https://github.com/mcu-tools/mcuboot/wiki/MCUboot-Project-Charter>`_ for more details.
+
+Content
+*******
+
+.. toctree::
+   :maxdepth: 1
+
+   release-notes.md
+   design.md
+   encrypted_images.md
+   imgtool.md
+   ecdsa.md
+   readme-zephyr.md
+   readme-ncs.rst
+   testplan-zephyr.md
+   release.md
+   SECURITY.md
+   SubmittingPatches.md
+
+Roadmap
+*******
+
+The issues being planned and worked on are tracked using GitHub issues. To give your input, visit `MCUboot GitHub Issues <https://github.com/mcu-tools/mcuboot/issues>`_.
+
+Browsing
+********
+
+You can find additional documentation on the bootloader in the source files.
+For more information, use the following links:
+* `boot/bootutil <https://github.com/mcu-tools/mcuboot/tree/main/boot/bootutil>`_: The core of the bootloader itself.
+* `boot/boot_serial <https://github.com/mcu-tools/mcuboot/tree/main/boot/boot_serial>`_: Support for serial upgrade within the bootloader itself.
+* `boot/zephyr <https://github.com/mcu-tools/mcuboot/tree/main/boot/zephyr>`_: Port of the bootloader to Zephyr.
+* `imgtool <https://github.com/mcu-tools/mcuboot/tree/main/scripts/imgtool.py>`_: A tool to securely sign firmware images for booting by MCUboot.
+* `sim <https://github.com/mcu-tools/mcuboot/tree/main/sim>`_: A bootloader simulator for testing and regression.
+
+Joining
+*******
+
+Developers are welcome!
+
+Use the following links to join or see more about the project:
+
+* `Our developer mailing list <https://groups.io/g/MCUBoot>`_
```

Comment:
`Our` > `MCUBoot official`?
Here and in line 65?

Reply:
fixed